hmm but based on Database
11:49:25 /home/jenkins/slave/workspace/1-pull-request-test/java/jtraverser/src/main/java/Database.java:251: error: method getNciFlag in class TreeNode cannot be applied to given types;
11:49:25 		return node.getNciFlag(flagMask);
you where using it as mask.. so there is another inconsistency i will see if there are more hint on how it was implemented before

--- a/java/devicebeans/src/main/java/mds/devices/Interface.java
+++ b/java/devicebeans/src/main/java/mds/devices/Interface.java
@@ -17,7 +17,7 @@ public interface Interface {
 	public float[] getFloatArray(String expr) throws Exception;
 	public double getDouble(String expr) throws Exception;
 	public String getString(String expr) throws Exception;
-	public String []getStringArray(String expr) throws Exception;
+	public String[] getStringArray(String expr) throws Exception;
 	public String getNodeName(int nid) throws Exception;
 	public String execute(String expr) throws Exception;
 	public void putDataExpr(int nid, String expr) throws Exception;

--- a/java/jtraverser/src/main/java/Database.java
+++ b/java/jtraverser/src/main/java/Database.java
@@ -239,16 +239,16 @@ public class Database implements Interface{
 		node.clearNciFlag(flagMask);
 
 	}
-	public void setNciFlag(int nid, int flagMask) throws Exception
+	public void setNciFlag(int nid, int flagId) throws Exception
 	{
 		MDSplus.TreeNode node = new MDSplus.TreeNode(nid, mdstree);
-		node.setNciFlag(flagMask);
+		node.setNciFlag(flagId);
 
 	}
-	public boolean getNciFlag(int nid, int flagMask) throws Exception
+	public boolean getNciFlag(int nid, int flagId) throws Exception
 	{
 		MDSplus.TreeNode node = new MDSplus.TreeNode(nid, mdstree);
-		return node.getNciFlag(flagMask);
+		return node.getNciFlag(flagId);
 	}
 	public int getNciFlags(int nid) throws Exception
 	{

